app/routes/download_results.py

```diff
@@ -0,0 +1,41 @@
+# Add the “download results” button to the web interface.
```

Comment:
Please, fix this docstring as well as described for the other module.

Reply:
Done